### PR TITLE
[Tests-Only] refactor share permission tests to not use skeleton files

### PR DESCRIPTION
**Pull request by dpakach**
## Description

refactor share permission tests to not use skeleton files

## Related Issue

- Related owncloud/QA#659

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPermissionsUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14553/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14584/

<p>




</p></details>

File: tests/acceptance/features/webUISharingNotificationsToRoot/shareWithGroups.feature [hunk 1]
```diff
@@ -17,7 +17,9 @@ Feature: Sharing files and folders with internal groups
     And user "Alice" has been added to group "grp1"
     And user "Brian" has been added to group "grp1"
     And user "Brian" has logged in using the webUI
-
+    Given user "Carol" has created folder "simple-folder"
+    Given user "Carol" has uploaded file "data.zip" to "data.zip"
+    Given user "Carol" has uploaded file "lorem.txt" to "simple-folder/lorem.txt"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingNotificationsToRoot/shareWithGroups.feature:22]:
```suggestion
    And user "Carol" has created folder "simple-folder"
    And user "Carol" has uploaded file "data.zip" to "data.zip"
    And user "Carol" has uploaded file "lorem.txt" to "simple-folder/lorem.txt"
```

File: tests/acceptance/features/webUISharingNotificationsToRoot/shareWithUsers.feature [hunk 2]
```diff
@@ -6,10 +6,14 @@ Feature: Sharing files and folders with internal users
...
     Given app "notifications" has been enabled
-    And these users have been created with default attributes:
+    And these users have been created with default attributes and without skeleton files:
       | username |
       | Alice    |
       | Brian    |
+    Given user "Alice" has created folder "simple-folder"
+    Given user "Alice" has created folder "simple-empty-folder"
+    Given user "Alice" has uploaded file "data.zip" to "data.zip"
+    Given user "Alice" has uploaded file "lorem.txt" to "simple-folder/lorem.txt"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingNotificationsToRoot/shareWithUsers.feature:16]:
```suggestion
    And user "Alice" has created folder "simple-folder"
    And user "Alice" has created folder "simple-empty-folder"
    And user "Alice" has uploaded file "data.zip" to "data.zip"
    And user "Alice" has uploaded file "lorem.txt" to "simple-folder/lorem.txt"
```

File: tests/acceptance/features/webUISharingPermissionToRoot/shareFileWithMultipleUsers.feature [hunk 3]
```diff
@@ -37,20 +38,20 @@ Feature: Sharing files with multiple internal users with different permissions
       | field       | value                |
       | uid_owner   | Alice                |
       | share_with  | Brian                |
-      | file_target | /lorem (2).txt       |
+      | file_target | /lorem.txt       |
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingPermissionToRoot/shareFileWithMultipleUsers.feature:41]: Indentation can be fixed here.

File: tests/acceptance/features/webUISharingPermissionToRoot/shareFileWithMultipleUsers.feature [hunk 4]
```diff
@@ -37,20 +38,20 @@ Feature: Sharing files with multiple internal users with different permissions
...
       | item_type   | file                 |
       | permissions | <actual-permissions> |
     And user "Carol" should have received a share with these details:
       | field       | value                |
       | uid_owner   | Alice                |
       | share_with  | Carol                |
-      | file_target | /lorem (2).txt       |
+      | file_target | /lorem.txt       |
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingPermissionToRoot/shareFileWithMultipleUsers.feature:48]: Indentation can be fixed here.

File: tests/acceptance/features/webUISharingPermissionToRoot/shareFolderWithMultipleUsers.feature [hunk 5]
```diff
@@ -37,20 +39,20 @@ Feature: Sharing folders with multiple internal users with different permissions
       | field       | value                |
       | uid_owner   | Alice                |
       | share_with  | Brian                |
-      | file_target | /simple-folder (2)   |
+      | file_target | /simple-folder   |
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingPermissionToRoot/shareFolderWithMultipleUsers.feature:42]: same as above

File: tests/acceptance/features/webUISharingPermissionToRoot/shareFolderWithMultipleUsers.feature [hunk 6]
```diff
@@ -37,20 +39,20 @@ Feature: Sharing folders with multiple internal users with different permissions
...
       | item_type   | folder               |
       | permissions | <actual-permissions> |
     And user "Carol" should have received a share with these details:
       | field       | value                |
       | uid_owner   | Alice                |
       | share_with  | Carol                |
-      | file_target | /simple-folder (2)   |
+      | file_target | /simple-folder   |
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingPermissionToRoot/shareFolderWithMultipleUsers.feature:49]: same as above

File: tests/acceptance/features/webUISharingPermissionToRoot/sharePermissionsFileGroup.feature [hunk 7]
```diff
@@ -39,18 +40,18 @@ Feature: Sharing files with internal groups with permissions
       | field       | value                |
       | uid_owner   | Alice                |
       | share_with  | grp2                 |
-      | file_target | /lorem (2).txt       |
+      | file_target | /lorem.txt       |
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingPermissionToRoot/sharePermissionsFileGroup.feature:43]: same as above

File: tests/acceptance/features/webUISharingPermissionToRoot/sharePermissionsFileGroup.feature [hunk 8]
```diff
@@ -39,18 +40,18 @@ Feature: Sharing files with internal groups with permissions
...
       | item_type   | file                 |
       | permissions | <actual-permissions> |
     And user "Carol" should have received a share with these details:
       | field       | value                |
       | uid_owner   | Alice                |
       | share_with  | Carol                |
-      | file_target | /lorem (2).txt       |
+      | file_target | /lorem.txt       |
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingPermissionToRoot/sharePermissionsFileGroup.feature:50]: same as above

File: tests/acceptance/features/webUISharingPermissionToRoot/sharePermissionsFoldersGroup.feature [hunk 9]
```diff
@@ -39,18 +41,18 @@ Feature: Sharing folders with internal groups with different roles and permissio
       | field       | value                |
       | uid_owner   | Alice                |
       | share_with  | grp2                 |
-      | file_target | /simple-folder (2)   |
+      | file_target | /simple-folder   |
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingPermissionToRoot/sharePermissionsFoldersGroup.feature:44]: same as above

File: tests/acceptance/features/webUISharingPermissionToRoot/sharePermissionsFoldersGroup.feature [hunk 10]
```diff
@@ -39,18 +41,18 @@ Feature: Sharing folders with internal groups with different roles and permissio
...
       | item_type   | folder               |
       | permissions | <actual-permissions> |
     And user "Carol" should have received a share with these details:
       | field       | value                |
       | uid_owner   | Alice                |
       | share_with  | Carol                |
-      | file_target | /simple-folder (2)   |
+      | file_target | /simple-folder   |
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingPermissionToRoot/sharePermissionsFoldersGroup.feature:51]: same as above

File: tests/acceptance/features/webUISharingPermissionsUsers/sharePermissionsUsers.feature [hunk 11]
```diff
@@ -7,10 +7,16 @@ Feature: Sharing files and folders with internal users with different permission
...
+    And these users have been created with default attributes and without skeleton files:
       | username |
       | Alice    |
       | Brian    |
+    Given user "Brian" has created folder "simple-folder"
+    Given user "Brian" has created folder "simple-empty-folder"
+    Given user "Brian" has created folder "simple-folder/simple-empty-folder"
+    Given user "Brian" has uploaded file "data.zip" to "data.zip"
+    Given user "Brian" has uploaded file "lorem.txt" to "lorem.txt"
+    Given user "Brian" has uploaded file "lorem.txt" to "simple-folder/lorem.txt"
```
Comment (swoichha) [inline tests/acceptance/features/webUISharingPermissionsUsers/sharePermissionsUsers.feature:19]:
```suggestion
    And user "Brian" has created folder "simple-folder"
    And user "Brian" has created folder "simple-empty-folder"
    And user "Brian" has created folder "simple-folder/simple-empty-folder"
    And user "Brian" has uploaded file "data.zip" to "data.zip"
    And user "Brian" has uploaded file "lorem.txt" to "lorem.txt"
    And user "Brian" has uploaded file "lorem.txt" to "simple-folder/lorem.txt"
```

**Review by swoichha — CHANGES_REQUESTED** [review]

**Review by swoichha — APPROVED** [review]
LGTM

**Review by individual-it — APPROVED** [review]
please resolve conflicts

**Pull request merged**